stream.go

```diff
@@ -0,0 +1,150 @@
+package tarantool
+
+import (
+	"fmt"
+	"time"
+
+	"gopkg.in/vmihailenco/msgpack.v2"
+)
+
+type TxnIsolationLevel uint64
```

Comment:
Is there any reason to use 64 bytes type since there are only four levels and it's not like there's gonna be a lot of new ones?

Reply:
You are right, changed:

https://github.com/tarantool/go-tarantool/blob/0531fcb8bc1eae2633c42f7a534c85766c76ae4f/stream.go#L11